### ISSUE TITLE
Optional chaining crashing connection

### ISSUE THREAD

**Issue by Emetrop** [zeplin/cli-connect-react-plugin#24]
If optional chaining is used in components running `zeplin connect` fails connecting to components with this error

```
Connecting components to Zeplin components failed.

    Error occurred while processing ./../index.tsx with @zeplin/cli-connect-react-plugin:

    did not recognize object of type "ChainExpression"
```

Seems it's issue of `react-docgen`

https://github.com/reactjs/react-docgen/issues/463

With new released version of `react-docgen` it should work.

**Comment by yuqu** [comment]
Hey Martin, thanks for reporting. The fix is released with v0.2.6 🎉

**Comment by yuqu** [comment]
The fix caused an issue with global installations https://github.com/zeplin/cli-connect-react-plugin/issues/24

I reverted the changes with v0.2.7, will be looking for a solution via CI setup.

**Comment by yuqu** [comment]
Now fixed with [v0.2.8](https://github.com/zeplin/cli-connect-react-plugin/releases/tag/v0.2.8).
